bookwyrm/settings.py

```diff
@@ -369,6 +369,7 @@
 OTEL_SERVICE_NAME = env("OTEL_SERVICE_NAME", None)
 
 TWO_FACTOR_LOGIN_MAX_SECONDS = 60
+TWO_FACTOR_LOGIN_VALIDITY_WINDOW = 2
```

Comment:
It would be better to pull this value in from `.env` so that it doesn't get wiped out when pulling in the latest changes:
```python
TWO_FACTOR_LOGIN_VALIDITY_WINDOW = env.int("TWO_FACTOR_LOGIN_VALIDITY_WINDOW")
```

And then add `TWO_FACTOR_LOGIN_VALIDITY_WINDOW` to `env.example` with a value of `2`.
This actually also applies to `TWO_FACTOR_LOGIN_MAX_SECONDS` I guess so maybe the same for that would be good.

Reply:
Done 🙂

Reply:
Mostly out of curiosity, how does this behave when the env file is missing those values? 

Our upgrading instructions don't really mention anything about new env file values, and this has the potential of locking people out of their accounts, including the admins and moderators 😬

If that is the case we could just add some sensible defaults in `settings.py` when reading from the environment, just in case they're not set.

Reply:
if the .env is missing the value and no default is given it should throw an error the moment you try to create or run migrations, restart bookwyrm, etc.